app/decorators/user_decorator.rb

```diff
@@ -10,7 +10,7 @@ def email_tag
   end
 
   def first_name
-    object.name.split(" ").first
+    source.name.split(" ").first if source.name.present?
```

Comment:
We'll need to add a fallback to first_name if the `source.name` can be `nil` since `app/views/authors/show.haml` relies on it in several strings.